sdk/opendp/whitenoise/synthesizers/mwem/mwem.py

```diff
@@ -50,6 +51,9 @@ def fit(self, data, categorical_columns=tuple(), ordinal_columns=tuple()):
             self.data = data.copy()
         else:
             raise ValueError("Data must be a numpy array.")
+
+        if self.split_factor > 0 and self.splits == []:

```

Comment:
Do we use split_factor as -1 or mostly check > 0 and avoid it? If so, it would be better to have None to not have unintended bugs

Reply:
good call